docs/cel2/notices/l2-migration.md

```diff
@@ -1,21 +1,95 @@
-# Preparing for the L2 migration
-
-This page outlines breaking changes related to the Celo L2 migration. This migration requires changes by node operators on the different Celo networks.
+# Preparing for the L2 hardfork
 
 :::info
-This page will be updated continuously with information on upgrade runbooks and timelines as they come.
+This page will be kept updated with key information about the hardfork.
 
 * The Alfajores testnet has been migrated on block **26384000**, September 26, 2024.
 * The Baklava testnet has been migrated on block **28308600**, February 20, 2025.
 * Mainnet will be migrated on block **31056500**, March 25, 2025.
 
 :::
 
-## For node operators
+Node operators should review this page along with [Migrating a Celo L1 Node](../operators/migrate-node.md) and [Running a Celo Node](../operators/run-node.md).
+
+Besides that, the two main steps to prepare for the hardfork are
+
+1. Upgrading L1 nodes
+2. Running a pre-migration (or choosing to use `snap` sync to start your L2 node)
```

Comment:
nit: unclear if this second step means
(a) an alternative to running the pre-migration is to use `snap` sync but in either case it has to be done before the migration
(b) an alternative to running the pre-migration is to use `snap` sync at the time of the migration, meaning the preparation ahead of time is reduced to just step (1)